doc/release/v2_3_70.md

```diff
@@ -11,6 +11,10 @@ Important Changes
 * YARP_math can no longer be built using GSL. The `CREATE_LIB_MATH_USING_GSL`
   option was removed. Only Eigen is supported. `FindGSL.cmake` is no longer
   installed.
+* 31/12/2016: `RemoteControlBoard` device is no longer compatible with `ControlBoardWrapper2`
```

Comment:
`31/12/2016` `30/07/2014`? Just forget about the dates next time, nobody cares... :laughing: